pkg/image/importer/importer.go

```diff
@@ -65,6 +66,7 @@ func NewImageStreamImporter(retriever RepositoryRetriever, maximumTagsPerRepo in
 		limiter:   limiter,
 
 		digestToRepositoryCache: make(map[gocontext.Context]map[manifestKey]*api.Image),
+		digestToLayerSizeCache:  make(map[gocontext.Context]map[string]int64),
```

Comment:
What's the purpose of grouping layer sizes per individual contexts? Context does not have any impact on `layer <-> size` mapping. Digest should have just one corresponding size.